modules/custom/az_finder/src/Plugin/better_exposed_filters/filter/AZFinderTaxonomyIndexTidWidget.php

```diff
@@ -358,6 +358,11 @@ public function preprocessAzFinderTaxonomyIndexTidWidget(array &$variables): voi
       $entity_type = 'taxonomy_term';
       $entity_id = is_numeric($child) ? $child : str_replace('tid:', '', $child);
       $state = $state_overrides[$entity_id] ?? $global_default_state;
+      // If the term should be removed, exclude it.
+      if ($state === 'remove') {
```

Comment:
I believe this needs to be updated to match the new value from AZFinderVocabulary.

```suggestion
      if ($state === 'hide') {
```